extensions/random-password-generator/src/Action.tsx

```diff
@@ -0,0 +1,12 @@
+import { ActionPanel, Action } from "@raycast/api";
+
+export function Actions({ password, setShowingDetails }: { password: string; setShowingDetails: () => void }) {
+  return (
+    <ActionPanel title={password}>
+      <ActionPanel.Section>
+        <Action.CopyToClipboard content={password} title="Copy Password" shortcut={{ modifiers: ["cmd"], key: "." }} />
+        <Action title="Toggle Detail" onAction={() => setShowingDetails()} />
```

Comment:
Could you assign an icon here? (Either all actions should have an icon or none of them, also see the [store guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store).)

Reply:
sure